exporter/signalfxexporter/translation/constants.go

```diff
@@ -220,5 +220,13 @@ translation_rules:
   mapping:
     free: df_inodes.free
     used: df_inodes.used
+    used: memory.used
```

Comment:
what about this line? looks like accidentally added?

Reply:
Yeah, deleted.